apps/web/modules/bookings/views/bookings-single-view.test.tsx

```diff
@@ -1,153 +0,0 @@
-import { render } from "@testing-library/react";
```

Comment:
removing this test file because it was testing `HeadSeo` component used in bookings-single-view component